src/python_be.cc

```diff
@@ -713,30 +732,58 @@ ModelInstanceState::ExecuteBLSRequest(std::shared_ptr<IPCMessage> ipc_message)
       }
 
       if (pb_exception.what() != nullptr) {
-        infer_response =
-            request_executor->Infer(infer_request, &inference_response);
+        auto response_future =
+            request_executor->Infer(infer_request, is_stream);
+        GetBLSResponses(infer_responses, std::move(response_future));
```

Comment:
We don't want to wait for all the responses to be received and them send them to the stub process. We want to send the responses as we are receiving them from the model.

Reply:
Synced offline that we will be addressing this comment in the next release. Created a [ticket](https://jirasw.nvidia.com/browse/DLIS-4529) for tracking.